lib/tasks/publishing_api.rake

```diff
@@ -251,6 +251,18 @@ namespace :publishing_api do
     end
   end
 
+  desc "Republish all statistics which have HTML attachments to the Publishing API"
+  task republish_stats_with_html_attachments: :environment do
+    document_ids = Publication
+      .publicly_visible
```

Comment:
Do we not want to republish draft HTML attachments too?

Reply:
thanks, we probably do! there's also a scope already set up for `statistical_documents` so i'll redraft this shortly.